store/tikv/region_cache.go

```diff
@@ -1759,19 +1781,31 @@ type Store struct {
 type resolveState uint64
 
 const (
+	// The store is just created and normally is being resolved.
+	// Store in this state will only be resolved by initResolve().
 	unresolved resolveState = iota
+	// The store is resolved and its address is valid.
 	resolved
+	// Request failed on this store and it will be re-resolved by asyncCheckAndResolveLoop().
 	needCheck
+	// The store's address or label is changed and marked deleted.
+	// There is a new store struct replaced it in the RegionCache and should
+	// call changeToActiveStore() to get the new struct.
 	deleted
+	// The store is a tombstone. Should invalidate the region if tries to access it.
+	tombstone
```

Comment:
does PD have the chance(or mechanism) to change a store state from tombstone to another status~? or it became a tombstone, it will always be a tombstone?

Reply:
It will always be a tombstone. I confirmed it from @nolouch.